packages/fiori/src/types/FCLLayout.ts

```diff
@@ -24,7 +26,7 @@ const FCLLayouts = {
 	 * @type {TwoColumnsStartExpanded}
 	 * @public
 	 */
-	TwoColumnsStartExpanded: "TwoColumnsStartExpanded",
+	TwoColumnsStartExpanded = "TwoColumnsStartExpanded",
 
 	/**
 	 * Desktop: 33/67/-  Start and Mid (expanded) columns are displayed
```

Comment:
Here we have slashes in the js doc for several options in the FCLLayout enum, check if all the options are generated properly in the fiori/dist/api.json, because slashes currently breaks the processing of the docs